app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PostsController < ApplicationController
```

Comment:
- [ ] Please kindly include the index and show actions for the PostsController and also include corresponding routes and view pages.
- [ ] Also please ensure that uses post.recent_comments to render 5 most recent comments in the `app/views/posts/index.html.erb` page
- [ ] Also, the Post show page displays posts with all comments

Reply:
@AnselemOdims , please note that this is assignment for day 3 and the instructions are as below

#### Implement the design from the sneak peek wireframes.

   Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)
  Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).